ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -129,11 +130,90 @@ def write_record(self, record: Record) -> None:
             self.write_ml_model(record)
         elif isinstance(record, OMetaDatabaseAndModel):
             self.write_dbt_models(record)
+        elif isinstance(record, OMetaDatabaseAndTableAndLocation):
+            self.write_tables_with_location(record)
         else:
             logging.info(
                 f"Ignoring the record due to unknown Record type {type(record)}"
             )
 
+    def write_tables_with_location(
+        self, db_and_table_and_location: OMetaDatabaseAndTableAndLocation
+    ):
+        try:
+            db_request = CreateDatabaseEntityRequest(
+                name=db_and_table_and_location.database.name,
+                description=db_and_table_and_location.database.description,
+                service=EntityReference(
+                    id=db_and_table_and_location.database.service.id,
+                    type="databaseService",
+                ),
+            )
+            db = self.metadata.create_or_update(db_request)
+            location_request = CreateLocationEntityRequest(
+                name=db_and_table_and_location.location.name,
+                description=db_and_table_and_location.location.description,
+                service=EntityReference(
+                    id=db_and_table_and_location.location.service.id,
+                    type="storageService",
+                ),
+            )
+            location = self.metadata.create_or_update(location_request)
+            table_request = CreateTableEntityRequest(
+                name=db_and_table_and_location.table.name,
+                tableType=db_and_table_and_location.table.tableType,
+                columns=db_and_table_and_location.table.columns,
+                description=db_and_table_and_location.table.description,
+                database=db.id,
+            )
+
+            if (
+                db_and_table_and_location.table.viewDefinition is not None
+                and db_and_table_and_location.table.viewDefinition != ""
```

Comment:
You could simplify this with the variable `truthiness`. `not None` and `!= ""` is equivalent to `if db_and_table_and_location.table.viewDefinition`

Reply:
+1 for above comment

Reply:
fixed